src/ontology/cl-edit.owl

```diff
@@ -31343,6 +31343,15 @@ AnnotationAssertion(rdfs:label obo:CL_4033037 "mucus secreting cell of tracheobr
 EquivalentClasses(obo:CL_4033037 ObjectIntersectionOf(obo:CL_0000319 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0007196) ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_8410077)))
 SubClassOf(obo:CL_4033037 obo:CL_0002202)
 
+# Class: obo:CL_4033049 (taste receptor cell of tongue)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1111/j.1096-3642.1981.tb01137.x") Annotation(oboInOwl:hasDbXref "PMID:10052456") Annotation(oboInOwl:hasDbXref "PMID:28655883") Annotation(oboInOwl:hasDbXref "http://www.nejohnston.org/Birds/documents/AvianTongues_Johnston.pdf") obo:IAO_0000115 obo:CL_4033049 "A taste receptor cell that is part of a taste bud of a tongue.")
```

Comment:
After a cursory review, I did not find this cell type described in "http://www.nejohnston.org/Birds/documents/AvianTongues_Johnston.pdf".